### ISSUE TITLE
Add support for animations

### ISSUE THREAD

**Issue by Narretz** [Pasvaz/bindonce#71]
Hey, I think bindonce should support animations when ngAnimate is included. This is necessary if you want to show something once, and need an animation there. Also, 1.2 is the current stable release, so support for 1.0.x can be removed.

**Comment by Pasvaz** [comment]
yes it is something I always wanted to do but I had no time for. I'll try to add it to the dev branch.

**Comment by Narretz** [comment]
Thanks, that'd be great!

**Comment by lonepalmus** [comment]
Hi - any update on this? Would love to use ngAnimate with bindonce
